### PR TITLE
Feature/add panel group

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: online-services-ui/src/app/shared/ui/panel-group/panel-group.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PanelGroupComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```
[the remark this answers is not in the document]
Comment (99999989) [reply online-services-ui/src/app/shared/ui/panel-group/panel-group.component.spec.ts]: It's actually just a generated test. I will delete it.

File: online-services-ui/src/app/shared/ui/panel-group/panel-group.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="arrow-container" *ngIf="!isFirstPanelGroup">
+  <div class="arrow-down" [class.related]="isRelatedPanelGroup"></div>
+</div>
+<div class="panel-group-content">
+  <ng-content select=".content"></ng-content>
+</div>
+<div class="panel-group-footer">
+  <ng-content select=".footer"></ng-content>
```
[the remark this answers is not in the document]
Comment (99999989) [reply online-services-ui/src/app/shared/ui/panel-group/panel-group.component.html]: The buttons are placed in the footer of the panel grop

File: online-services-ui/src/app/shared/ui/panel/panel.component.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<div class="arrow-container" *ngIf="!isFirstPanel">
```
[the remark this answers is not in the document]
Comment (99999989) [reply online-services-ui/src/app/shared/ui/panel/panel.component.html]: It's needed to know whether to display an arrow at the start. It's need, but I think about renaming it.

File: online-services-ui/src/app/shared/ui/panel/panel.component.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  <div class="arrow-down" [class.related]="isRelatedPanel"></div>
+</div>
+<div class="panel mat-body">
```
[the remark this answers is not in the document]
Comment (99999989) [reply online-services-ui/src/app/shared/ui/panel/panel.component.html]: removed

File: online-services-ui/src/app/shared/ui/panel/panel.component.scss [hunk 5]
```diff
@@ -0,0 +1,27 @@
+@import "../../../../colors";
+.panel {
+  background-color: $color-light-grey;
+  box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.35);
+  padding: 15px 20px 10px;
+  margin: 0.25em;
+  ::ng-deep .panel-title {
```
Comment (ganqqwerty) [inline online-services-ui/src/app/shared/ui/panel/panel.component.scss]: do you really need ng-deep here? Panel-title doesn't typically have subcomponents

File: online-services-ui/src/app/shared/components/panel-group/panel-group.component.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'os-panel-group',
+  templateUrl: './panel-group.component.html',
+  styleUrls: ['./panel-group.component.scss', '../panel/panel-arrow.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```
Comment (ganqqwerty) [inline online-services-ui/src/app/shared/components/panel-group/panel-group.component.ts]: please add the comment explaining why you use onPush strategy

File: online-services-ui/src/app/shared/components/panel/panel.component.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'os-panel',
+  templateUrl: './panel.component.html',
+  styleUrls: ['./panel.component.scss', './panel-arrow.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```
Comment (ganqqwerty) [inline online-services-ui/src/app/shared/components/panel/panel.component.ts]: same here

File: online-services-ui/src/app/forms/avp-form/employments/employments.component.html [hunk 8]
```diff
@@ -1,57 +1,83 @@
...
-                     formControlName="selfEmployed"></os-yes-no-input>
+    <os-panel [showArrow]="true">
+      <os-yes-no-input question="Have you been self-employed in March 2018?"
+                       formControlName="selfEmployed"></os-yes-no-input>
+    </os-panel>
 
-    <os-files-upload *ngIf="form.value.selfEmployed"></os-files-upload>
+    <os-panel [showArrow]="true" [colorizeArrow]="true" *ngIf="form.value.selfEmployed">
+      <div class="panel-title">
+        Bitte hängen Sie Scans von Belegen und Abrechnungen an.<span>Optional</span></div>
```
Comment (ivanmrva) [inline online-services-ui/src/app/forms/avp-form/employments/employments.component.html]: every other message besides this one is here in english

File: online-services-ui/src/app/shared/components/panel-group/panel-group.component.ts (hunk 6 above)
Comment (99999989) [reply online-services-ui/src/app/shared/components/panel-group/panel-group.component.ts]: done

File: online-services-ui/src/app/shared/components/panel/panel.component.ts (hunk 7 above)
Comment (99999989) [reply online-services-ui/src/app/shared/components/panel/panel.component.ts]: done

**Pull request merged**